meilisearch-schema/src/lib.rs

```diff
@@ -36,7 +37,8 @@ impl Into<u16> for IndexedPos {
     }
 }
 
-#[derive(Serialize, Deserialize, Debug, Copy, Clone, Default, PartialOrd, Ord, PartialEq, Eq, Hash)]
+#[derive(Serialize, Deserialize, Debug, Copy, Clone, Default, PartialOrd, Ord, PartialEq, Eq, Hash, AsBytes, FromBytes)]
```

Comment:
Hum... Ok, I think we should revert these changes and only keep the serde traits as we will use an `HashSet` instead of a slice.